cmd/soroban-cli/src/utils.rs

```diff
@@ -161,6 +161,41 @@ pub fn get_name_from_stellar_asset_contract_storage(storage: &ScMap) -> Option<S
     }
 }
 
+pub mod http {
```

Comment:
Was this added by accident?

Reply:
Hmm no, I think you are looking at the merge commit. It's not in the [diff](https://github.com/stellar/stellar-cli/pull/1628/files#diff-643bf04585e9188dfeb0f09377be2943d1b4ed23326b6ff31344035a9e7ef592)